tesseract_geometry/include/tesseract_geometry/mesh_parser.h

```diff
@@ -157,10 +157,10 @@ std::vector<std::shared_ptr<T>> extractMeshData(const aiScene* scene,
     {
       aiMaterial* mat = scene->mMaterials[a->mMaterialIndex];
       {
-        Eigen::Vector4d base_color;
+        Eigen::Vector4d base_color = Eigen::Vector4d::Zero();
         double metallic = 0.0;
         double roughness = 0.5;
-        Eigen::Vector4d emissive;
+        Eigen::Vector4d emissive = Eigen::Vector4d::Zero();
```

Comment:
I think these both need to be initialized to make clang-tidy happy.

Reply:
I was getting failures with the GCC compiler in conda. I guess it has stricter checks.